tegral-openapi/tegral-openapi-dsl/src/main/kotlin/guru/zoroark/tegral/openapi/dsl/PathDsl.kt

```diff
@@ -215,8 +215,11 @@ class PathBuilder(private val context: OpenApiDslContext) : PathDsl, Buildable<P
         }
     override val parameters: MutableList<Buildable<Parameter>>
         get() = error(WRITE_ONLY_ERROR_MSG)
-    override val securityRequirements: MutableList<SecurityRequirement>
+    override var securityRequirements: MutableList<SecurityRequirement>
         get() = error(WRITE_ONLY_ERROR_MSG)
+        set(value) {
+            addOperationDefault { securityRequirements = value }
+        }
```

Comment:
Having a `var` `MutableList` looks a bit weird to me, do you have an example on how this could be used in the DSL?